web/channels/stream_channel.ex

```diff
@@ -8,7 +8,11 @@ defmodule Juicebox.StreamChannel do
 
   def join("stream:" <> stream_id, _params, socket) do
     PubSub.subscribe(Juicebox.PubSub, "juicebox:stream:server:" <> stream_id)
+    IO.inspect Juicebox.Stream.Supervisor.start_stream(stream_id)
     {:ok, queue} = Stream.queue(stream_id)
+
+    send(self, {:after_join, stream_id})
+
     {:ok, %{ queue: queue }, socket}
```

Comment:
Ah wait, I saw you removed the connect action on the client. We don't need to send the queue here in that case, just return `{:ok, socket}`